BitwardenShared/UI/Vault/Vault/VaultList/VaultListView.swift

```diff
@@ -69,30 +69,44 @@ private struct SearchableVaultListView: View {
 
     /// A view that displays the empty vault interface.
     @ViewBuilder private var emptyVault: some View {
-        GeometryReader { reader in
-            ScrollView {
-                VStack(spacing: 24) {
-                    vaultFilterRow
-                        .padding(.top, 16)
-
-                    Spacer()
-
-                    Text(Localizations.noItems)
-                        .multilineTextAlignment(.center)
-                        .styleGuide(.callout)
-                        .foregroundColor(Asset.Colors.textPrimary.swiftUIColor)
-
-                    Button(Localizations.addAnItem) {
-                        store.send(.addItemPressed)
-                    }
-                    .buttonStyle(.tertiary())
+        VStack(spacing: 24) {
+            vaultFilterRow
+                .padding(.top, 16)
+
+            Spacer()
+
+            Image(decorative: Asset.Images.emptyVaultItems)
+                .resizable()
+                .frame(width: 100, height: 100)
+
+            VStack(spacing: 12) {
```

Comment:
🎨 It looks like this `VStack` should switch to an `HStack` in landscape, should we do that as part of this? Would it work to use `PageHeaderView` here?

https://www.figma.com/design/eH8ybfmeq726N1SyvcoK2K/Bitwarden-iOS-Mobile-App-Reskin---V3-(current)?node-id=13870-311465&t=8HWibq0igAUVoBqk-4